packages/react/src/elements/components/Toast/toast.scss

```diff
@@ -21,18 +21,34 @@
 
 .hig__toast--primary {
   border-color: color(hig-blue-50);
+
+  .hig__toast__image-container svg {
+    fill: color(hig-blue-50);
+  }
 }
 
 .hig__toast--success {
   border-color: color(hig-green-good);
+
+  .hig__toast__image-container svg {
+    stroke: color(hig-green-good);
```

Comment:
Seems like we need to standardize whether icons are drawn as strokes or fills...

Reply:
I think we've asked the designers to export icons using only fills so we can use CSS to change icon colors.